ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -723,3 +723,16 @@ def _unsigned_int_bytes_repr(item_val, /, *, dtype=None):
             return b"".join(bytes_reprs)
         else:
             raise ValueError("Unsupported data type for the array.")
+
+    def var(
+        self, axis=None, dtype=None, out=None, ddof=0, keepdims=False, *, where=True
+    ):
+        return np_frontend.var(
+            self,
+            axis=axis,
+            dtype=dtype,
+            out=out,
+            ddof=ddof,
+            keepdims=keepdims,
+            where=where,
+        )
```

Comment:
Looks like the tests are failing because they are unable to locate this function (https://github.com/unifyai/ivy/actions/runs/6200453935/job/16835349713?pr=22336#step:3:10495). Could you move this function at bit higher up in the file where other numpy nd.array instance methods are implemented?